css/theme-lava.css

```diff
@@ -622,7 +622,6 @@ nav .container {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   content: "\e688";
-  color: #fff;
   font-size: 24px;
```

Comment:
You have just removed the color property but the default color remains the same i.e.white so No change in UI

Reply:
Thank you @mohitm15 and @Chaitanyassr for reviewing my pull request! :)
According to my understanding, the pull request made resolves the issue raised #672 

I have attached screenshots and highlighted with a red rectangle, to show reflected changes. 

Before
![Screenshot (94)](https://user-images.githubusercontent.com/50296407/76000210-60c45980-5f29-11ea-94c2-136489907118.png)

After 
![Screenshot (103)](https://user-images.githubusercontent.com/50296407/76001119-a3d2fc80-5f2a-11ea-9696-43eb73ad535b.png)

I wanted to ask instead of deleting the color property, should I set it to black and update my pull request?

Reply:
Change the color accordingly for that particular icon or class instead of deleting it